test/test-utils.sh

```diff
@@ -78,6 +78,25 @@ else
     STAT_BIN=(stat)
 fi
 
+function find_xattr() {
+    local FIND_RESULT=0
+
+    set +o errexit
```

Comment:
What part of this needs to manipulate errexit?  Can't you just do something like:
```
getfattr --absolute-names -n "$1" "$2" >/dev/null 2>&1 || return 1
```